src/test/scala/org/kibanaLoadTest/helpers/ESClient.scala

```diff
@@ -48,17 +48,24 @@ class ESClient(config: ESConfiguration) {
     try {
       logger.info(s"Ingesting to stats cluster: ${jsonList.size} docs")
       jsonList.par.foreach(json => {
-        client.index(
-          new IndexRequest(indexName)
-            .source(json.toString(), XContentType.JSON),
-          RequestOptions.DEFAULT
-        )
+        try {
+          client.index(
```

Comment:
This looks like a single request.  Not a problem, but I did run into performance issues indexing a log of data this way, in NodeJS.  I ended up switching to a bulk call. 
Not sure that's needed here, but I figured I'd mention it :)

Eg. https://github.com/elastic/kibana/blob/master/src/dev/code_coverage/ingest_coverage/ingest.js#L27

Not even sure the Java Client has that, or if you need it.  lol

Reply:
Nice! I will play with it in separate PR to be on the safe page.